pages/user guides/exporting_move_groups_to_csv.md

```diff
@@ -0,0 +1,58 @@
+---
+toc: false
+title: Exporting servers from a Move Group to CSV
+keywords: move_groups, servers, csv, rivermeadow, export
+last_updated: April 17, 2019
+summary: "Exporting servers from a move group to format that can be used by Rivermeadow's bulk update."
+sidebar: user_guidesbar
+folder: userguides
+permalink: exporting-servers-to-csv.html
+---
+
+Here we will show how to take an entire migration wave of servers from Tidal Migrations and export them for direct ingestion into Rivermeadow, to perform a VM migration.
+
+# Requirements
+
+This guides assumes you have access to a Linux like environment, you will need:
+
+-   Posix compliant shell
+-   [Curl](https://curl.haxx.se/download.html)
+-   [jq](https://stedolan.github.io/jq/)
+-   [A copy of our move group script](https://github.com/tidalmigrations/gists/blob/master/move_group_to_csv.sh)
+
+
+# Steps
+
+
+## 1. Authentication information
+
+To authenticate to your account, we'll need to collect the subdomain, username, password. If you don't know what your subdomain is you can find it here:
+
+{% include image.html file="subdomain.png" caption="An image highlighting how to find your organization's subdomain." %}
+![img](./subdomain.png)
+
+Once you have gathered this information, you'll need set it as an environment variable, substitute in your information to the commands below and run them in your shell:
+
+    export username="<your username>"
```

Comment:
Not sure how this renders but _maybe consider_ using back ticks either single or triple for multiline blocks around the commands to be run.